src/cli.rs

```diff
@@ -26,6 +26,25 @@ pub enum ReconstructSource {
 
 #[derive(Subcommand)]
 pub enum Command {
+    /// Download L2 state from L1 to JSON file.
+    #[command(hide = true)]
+    Download {
+        /// The Ethereum JSON-RPC HTTP URL to use.
+        #[arg(long)]
+        http_url: String,
+        /// Ethereum block number to start state import from.
+        #[arg(long, default_value_t=ethereum::GENESIS_BLOCK)]
+        start_block: u64,
+        /// The number of blocks to filter & process in one step over.
+        #[arg(long, default_value_t=ethereum::BLOCK_STEP)]
+        block_step: u64,
```

Comment:
```suggestion
        #[command(flatten)]
        l1_fetcher_options: L1FetcherOptions,
```
We could move these common L1 fetcher arguments into a separate `struct`, and then use [`#[command(flatten)]`](https://docs.rs/clap/latest/clap/_derive/index.html#flattening-hand-implemented-args-into-a-derived-application) to add them to the command.